packages/victory-scatter/src/victory-scatter.test.tsx

```diff
@@ -239,7 +239,7 @@ describe("components/victory-scatter", () => {
               ariaLabel={({ datum }) =>
                 `scatter point x: ${datum.x}, y:${datum.y}`
               }
-              tabIndex={({ index }) => index + 10}
+              tabIndex={({ index }) => (index as number) + 10}
```

Comment:
sharing my own learning from a recent PR: since index here has a type of `string | number`, instead of explicitly casting to a number, it's safer to do a runtime conversion with `+index` here

Reply:
Ah, thanks for this. Do you have an example of what you mean here?

Reply:
Yup! Here's what I ended up doing for a similar case: https://github.com/FormidableLabs/victory/pull/2387/files#diff-59245dd3946a6d7b3f4c439108b380ccb32954c2a205ecef8f02dcaa7369d0a0R162

Reply:
Oh! Haha that's super clear, awesome! Thank you so much, I'll do this!